pandas/tests/tools/test_to_datetime.py

```diff
@@ -2773,3 +2773,23 @@ def test_to_datetime_monotonic_increasing_index(cache):
     result = to_datetime(times.iloc[:, 0], cache=cache)
     expected = times.iloc[:, 0]
     tm.assert_series_equal(result, expected)
+
+
+@pytest.mark.parametrize(
+    "input",
+    [
+        Series(
```

Comment:
1. Could you parameterize over `[40, start_caching_at]`?
2. Since the Series contain the same data essentially, could you move the construction in the test body?